magicctapipe/io/io.py

```diff
@@ -1032,24 +876,36 @@ def load_dl2_data_file(input_file, quality_cuts, event_type, weight_type_dl2):
         If the input event type is not known
     """
 
+    TEL_NAMES, TEL_COMBINATIONS = telescope_combinations(config)
```

Comment:
the same 6 lines are also in load_mc_dl2_data_file
better to extract them in a separate function to make sure that the same code is used for data and MCs

Reply:
the only difference is that in real data we do not ask for magic_stereo == True in the 'software' because standard MAGIC + LST-1 data are always collected with MAGIC stereo trigger, but we can take this into account easily

Reply:
fixed